src/renderer/components/InstanceModal/SavesTabServer.ts

```diff
@@ -0,0 +1,89 @@
+import { readInfo, ServerInfo } from "@xmcl/server-info";
+import fs from "fs-extra";
+import path from "path";
+import Instance from "../../Instance";
+
+import { queryStatus, Status, QueryOptions } from "@xmcl/client";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['QueryOptions' is defined but never used.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5263046)